pkg/kubelet/cm/container_manager.go

```diff
@@ -25,7 +25,7 @@ type ContainerManager interface {
 	// Runs the container manager's housekeeping.
 	// - Ensures that the Docker daemon is in a container.
 	// - Creates the system container where all non-containerized processes run.
-	Start() error
+	Start(*api.Node) error
```

Comment:
+1 for taking in just the information that is necessary, ideally a separate object.